### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by AlbertoElias**
This will be merged into the beta branch

### CODE REVIEW

File: test/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		secondaryNav.classList.add('o-header__nav--secondary-theme');
+		anotherHeaderEl.appendChild(secondaryNav);
+
+		var toolsNav = document.createElement('nav');
+		toolsNav.setAttribute('data-o-component', 'o-hierarchical-nav');
+		toolsNav.classList.add('o-header__nav--tools-theme');
+		anotherHeaderEl.appendChild(toolsNav);
+
+		var myHeader = new Header(anotherHeaderEl);
+		expect(anotherHeaderEl.getAttribute('data-o-header--js')).to.not.be(null);
```
Comment (kaelig) [inline test/Header.test.js]: Is this testing that this line below worked?

```js
anotherHeaderEl.setAttribute('data-o-component', 'o-header');
```

Not sure testing any `setAttribute` was applied helps us massively. Should it be checked at all?

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+		// frameworks to use
+		// available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+		frameworks: ['mocha', 'browserify'],
+
+
+		// list of files / patterns to load in the browser
+		files: [
+			'http://polyfill.webservices.ft.com/v1/polyfill.js?ua=safari/4',
```
Comment (kaelig) [inline karma.conf.js]: Should we mention in a comment that it's for PhantomJS that we load this variant of the polyfill?

**Comment by kaelig** [comment]
:+1: Well done!

File: karma.conf.js (hunk 2 above)
Comment (AlbertoElias) [inline karma.conf.js]: tbh, all the comments here are generated automatically, and that's the only line I've added. I haven't added that comment in the tests elsewhere, but might be a good idea to start adding it

Comment (kaelig) [inline karma.conf.js]: Ok cool.

Are we planning on updating to PhantomJS 2? Apparently it bundles a much more recent version of WebKit and maybe it'd fix that.

Comment (AlbertoElias) [inline karma.conf.js]: It will probably fix it and we will update. I didn't know they had finally released out. Wow, Node 0.12 and PhantomJS 2.0

Comment (kaelig) [inline karma.conf.js]: I know, right? The future is here.

**Pull request merged**